src/validation.ts

```diff
@@ -0,0 +1,76 @@
+import addFormats from 'ajv-formats';
+import Ajv, { ValidateFunction } from 'ajv';
+
+import workflowStepTemplate from './workflow_step_template_schema.json';
+import workflowTemplate from './workflow_template_schema.json';
+
+interface Options {
+  schema?: 'workflowTemplate' | 'workflowStepTemplate';
+}
```

Comment:
If I put these in a dedicated `types.ts` file, the compiler would generate two completely useless JS files, `types.js` and `types.js.map`. There is an option to skip `d.ts` files, which would not work here because the `ValidateFunction` import turns it into a module. In a larger project this would generate potentially dozens of meaningless files from TS files that contain nothing but type declarations. How would one avoid that? 🤔

Reply:
For the middleware, we pass in [emitDeclarationOnly](https://www.typescriptlang.org/tsconfig#emitDeclarationOnly) option to tsc.. so tsc responsible only to do type checking and emitting d.ts file, while babel is responsible for actually [compiling the files to javascript](https://github.com/labforward/laboperator-middleware-development/blob/develop/src/cli/compile.js#L7)..

Reply:
Meh, I don't want to add Babel just for that. This looks to me like a problem that basically almost every project has, seems weird that there is no built-in way to handle that... 🤔

Reply:
And does that mean that Babel would be able to detect files that have no code worth compiling?

Reply:
Babel doesn't care about `d.ts` file.. I guess that's the issue? I just realized that you named the dedicated types file as `types.ts`, while it should actually be `types.d.ts`?